docs/core/extensions/generic-host.md

```diff
@@ -136,8 +132,83 @@ The configuration created by `ConfigureAppConfiguration` is available in [HostBu
 
 For more information, see [Configuration in .NET](configuration.md).
 
+## Host shutdown
+
+A hosted service process can be stopped in the following ways:
+
+1. If someone doesn't call <xref:Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run%2A> or <xref:Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.WaitForShutdown%2A?displayProperty=nameWithType> and the app exits normally with `Main` completing.
```

Comment:
This should be a bulleted list since it's not sequential.

Reply:
```suggestion
- If someone doesn't call <xref:Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run%2A> or <xref:Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.WaitForShutdown%2A?displayProperty=nameWithType> and the app exits normally with `Main` completing.
```